app/code/Magento/CustomerGraphQl/Model/Resolver/CreateCustomer.php

```diff
@@ -88,6 +89,8 @@ public function resolve(
             throw new GraphQlInputException(__($e->getMessage()));
         } catch (InputMismatchException $e) {
             throw new GraphQlInputException(__($e->getMessage()));
+        } catch (InputException $e) {
```

Comment:
Looks like we can process just `LocalizedException`
Also, do'nr forget to pass previous exception as parameter of new exception
`throw new GraphQlInputException(__($e->getMessage()), $e);`
Thanks

Reply:
Hi @naydav, thanks for the suggestion. 
Modified `CreateCustomer.php` file as per your instruction, please take a look.